frontend/consts_gen.py

```diff
@@ -162,7 +162,7 @@ def GenCharLookup(func_name, lookup, f, required=False):
 
   for char_code in sorted(lookup):
     f.write("  case '%s':\n" % CChar(char_code))
-    f.write('    return StrFromC("%s");\n' % CChar(lookup[char_code]))
+    f.write('    return StrFromC("%s", 1);\n' % CChar(lookup[char_code]))
```

Comment:
I think if you merge master into your branch, then Github won't show these diffs?

That might be a good idea since the change has been open for awhile